test/exceptions.py

```diff
@@ -170,7 +170,7 @@ def raise_testexception(msg, *args, **kwargs):
         self.assertErrorRegex(LoggedException, 'BOOM', raise_testexception, 'BOOM')
         logToFile(tmplog, enable=False)
 
-        log_re = re.compile(r"^%s :: BOOM \(at /.*/vsc/utils/testing.py:[0-9]+ in assertErrorRegex\)$" % rootlogname)
```

Comment:
yes, when in an egg, the printed path is a relative directory like `build/.../egg/...`

this is the error I was getting:

```
======================================================================
FAIL: test_loggedexception_location (test.exceptions.ExceptionsTest)
Test inclusion of location information in log message for LoggedException.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "build/bdist.macosx-10.10-intel/egg/vsc/test/exceptions.py", line 175, in test_loggedexception_location
    self.assertTrue(log_re.match(logtxt), "%s matches %s" % (log_re.pattern, logtxt))
AssertionError: ^runpy :: BOOM \(at /.*/vsc/utils/testing.py:[0-9]+ in assertErrorRegex\)$ matches runpy :: BOOM (at build/bdist.macosx-10.10-intel/egg/vsc/utils/testing.py:77 in assertErrorRegex)
```